bin/weewx/drivers/vantage.py

```diff
@@ -1481,6 +1502,14 @@ def _archive_datetime(datestamp, timestamp):
                       0, 0, -1)                     # have OS guess DST
         # Convert to epoch time:
         ts = int(time.mktime(time_tuple))
+        # fix for hour mismatches at begin of summertime
+        # calculate a time tuple from the UTC time stamp
+        utc_time_tuple = time.localtime(ts)
```

Comment:
Ok, I now got what you mean:
The name of the variable should be something like: 'vantage_local_time_tuple' instead of 'utc_time_tuple'.